Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build compilation image
-FROM ruby:3.1.1-alpine3.15 as builder
```

Comment:
I think this should reference `ruby:3.1-alpine3.15` - ie _not_ specify the patch level - that way we automatically benefit from patch level updates without doing any accidental ruby version bumps